pkg/activator/net/revision_backends.go

```diff
@@ -73,6 +73,7 @@ type revisionWatcher struct {
 	rev      types.NamespacedName
 	protocol networking.ProtocolType
 	updateCh chan<- revisionDestsUpdate
+	finished chan struct{}
```

Comment:
These types of channels are pretty much always named `done`.
Please consider renaming it for consistency.

Reply:
Makes sense. How does renaming `doneCh` -> `stopCh` and `finished` -> `done` in `revisionWatcher` sound?